libs/vuepress/src/executors/browser/executor.ts

```diff
@@ -0,0 +1,29 @@
+import { ExecutorContext } from '@nrwl/devkit';
+import * as path from 'path';
+import { BrowserExecutorSchema } from './schema';
+import { isVuepress2 } from '../../utils';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { build } = require('vuepress');
```

Comment:
I saw that you can use `import vuePress = require('vuepress')`. Kinda weird but you don't have to add the eslint ignore.